config/schemas/6/rapidast_schema.json

```diff
@@ -0,0 +1,605 @@
+                                    }
+                                },
+                                "required": [
+                                    "fileName"
+                                ]
+                            },
+                            "graphql": {
+                                "type": "object"
+                            },
+                            "spiderAjax": {
+                                "type": "object"
+                            },
+                            "replacer": {
+                                "type": "object",
+                                "properties": {
+                                    "rules": {
+                                        "type": "array",
+                                        "items": {
+                                            "type": "object",
+                                            "$comment": "@TODO: There are other properties, but they are passed directly to the Zap automation framework. These are the ones that have implications in Rapidast",
+                                            "properties": {
+                                                "matchRegex": {
+                                                    "type": "boolean"
+                                                },
+                                                "tokenProcessing": {
+                                                    "type": "boolean"
+                                                }
+                                            }
+                                        }
+                                    },
+                                    "parameters": {
+                                        "type": "object",
+                                        "properties": {
+                                            "deleteAllRules": {
+                                                "type": "boolean"
+                                            }
+                                        }
+                                    }
+                                }
+                            },
+                            "passiveScan": {
+                                "type": "object",
+                                "properties": {
+                                    "disabledRules": {
+                                        "type": "string"
+                                    }
+                                }
+                            },
+                            "activeScan": {
+                                "type": "object",
+                                "properties": {
+                                    "policy": {
+                                        "type": "string"
+                                    }
+                                }
+                            }
+                        },
+                        "required": [
+                            "apiScan"
```

Comment:
@jeremychoi I’m not entirely sure if this is correct. Is it possible to have a ZAP scanner section without using `apiScan`? If so, what conditions should apply in this case?

Reply:
One option for now is, one of [spider, spiderAjax, apiScan, importUrlsFromFile, graphql].  For the long term, ideally this could be grouped under a new key such as 'scan_scope'.

Reply:
Fixed